pkg/gen/input/workflows/internal/file/check_values_schema.yaml.template

```diff
@@ -1,5 +1,5 @@
 {{{{ .Header }}}}
-name: 'Check if values schema file has been updated'
+name: 'Values and schema'
```

Comment:
Opted for a short title here, as it is displayed in one lone together with the check name:

Values and schema / validate values.yaml against values.schema.json